content/references/protocol-reference/transactions/transaction-metadata.md

```diff
@@ -68,6 +68,15 @@ A `ModifiedNode` object contains the following fields:
 
 **Note:** If the modified ledger object has `PreviousTxnID` and `PreviousTxnLgrSeq` fields, the transaction always updates them with the transaction's own identifying hash and the index of the ledger version that included the transaction, but these fields' new value is not listed in the `FinalFields` of the `ModifiedNode` object, and their previous values are listed at the top level of the `ModifiedNode` object rather than in the nested `PreviousFields` object.
 
+## NFT Fields
+
+An `NFT` transaction response can contain the following fields:
```

Comment:
Putting `NFT` in code font makes it seem like you're referring to a specific thing with that exact name, but there isn't one. Maybe soften it up, like so:

```suggestion
Transactions involving NFTs can contain the following fields in the metadata:
```

Also, "can contain" is vague. Is it possible to be more specific about when it does and doesn't?

Based on these fields being lower-case, I am guessing that they are "non-canonical" fields that are calculated at the API layer and are not part of the hashed metadata—is that right? And are they returned by P2P `rippled` servers or only by Clio servers? These details are good to know.

Reply:
These fields are in `rippled@1.11`